### ISSUE TITLE
1.2.5 AttributeError: 'Namespace' object has no attribute 'x'

### ISSUE THREAD

**Issue by ilovezfs** [Alexis-benoist/eralchemy#49]
With 1.2.5
```
bash-3.2$ eralchemy -v
Traceback (most recent call last):
  File "/usr/local/bin/eralchemy", line 11, in <module>
    load_entry_point('ERAlchemy==1.2.5', 'console_scripts', 'eralchemy')()
  File "/usr/local/Cellar/eralchemy/1.2.5/libexec/lib/python2.7/site-packages/eralchemy/main.py", line 37, in cli
    check_args(args)
  File "/usr/local/Cellar/eralchemy/1.2.5/libexec/lib/python2.7/site-packages/eralchemy/helpers.py", line 17, in check_args
    args.x is not None:
AttributeError: 'Namespace' object has no attribute 'x'
bash-3.2$   
```
1.1.0 is fine.

**Comment by Alexis-benoist** [comment]
Thanks for the issue, I fixed it in 1.2.6

**Comment by ilovezfs** [comment]
Thanks @Alexis-benoist!
